src/commands/test/utils.py

```diff
@@ -9,6 +9,16 @@ def collect_subdirectories(root_dir: Path) -> List[str]:
     return [x[0] for x in os.walk(root_dir)]
 
 
+def collect_immediate_subdirectories(root_dir: Path) -> List[str]:
```

Comment:
Being honest i am not sure what this function does, could you rethink the naming?